index.js

```diff
@@ -76,7 +84,7 @@ function parseImportStatement(regExpArgs, config) {
     return '';
 }
 
-function isInternalImport(path, {internalImportPaths = []}) {
+function isInternalImport(path, { internalImportPaths = [] }) {
```

Comment:
is it possible to configure this?

Reply:
This one yes https://prettier.io/docs/en/options.html#bracket-spacing